core/src/main/java/bio/terra/pearl/core/model/datarepo/Dataset.java

```diff
@@ -17,5 +17,6 @@ public class Dataset extends BaseEntity {
     private UUID studyEnvironmentId;
     private UUID datasetId;
     private String datasetName;
+    private String status;
```

Comment:
Should we go ahead and make a a simple enum for this?

Reply:
Made it an enum 78aa8c84